src-electron/generator/helper-zcl.js

```diff
@@ -241,7 +242,18 @@ async function first_unused_enum_value(options) {
       }
     } while (isPresent)
   }
-  return unusedValue
+
+  if (format == 'hex') {
+    let out = unusedValue.toString(16)
+    if (this.size && this.size > 0) {
+      while (out.length < this.size * 2) {
```

Comment:
What is this.size going to be in practice?

Reply:
Number of bytes of the enum. This just pads the hex with zeros, so you get '01' instead of '1', so you can have nicely aligned values.

If you don't want this logic, let me know and I'll delete it.